src/render-html/css.js

```diff
@@ -11,6 +11,7 @@ html {
   -moz-user-select: none; /* Firefox 2+ */
   -ms-user-select: none; /* IE 10+ */
   user-select: none; /* Standard syntax */
+  -khtml-user-select: none;
```

Comment:
We should probably do something like the js.js to es3.js conversion, for example, Autoprefixer.
Are you sure khtml is relevant here?
`KDE Konqueror / Apple Safari 1.1 through Safari 2.0, WebKit-Based Browsers`

Reply:
Maybe UIWebView is so old? This one is relevant I guess: #1716

Reply:
hmm
So until we use `WKWebView` to fix this we can take this PR.
I have tested it, it fixes issues. We can include this in beta & definitely improve in future :)

Reply:
Have a look at the linked PR in React Native and maybe test/comment so they can get more incentive to merge in the main project ;)